.github/workflows/cypress-workflow.yml

```diff
@@ -24,12 +25,12 @@ jobs:
         with:
           # TODO: Parse this from security analytics plugin
```

Comment:
[Not blocking] Do we know what needs to be done here? Can you create a task with details?

Reply:
This would essentially involve an enhancement which would allow us to retrieve a list of the supported JDKs from the backend rather than maintaining a list of JDKs in the front and backend repos. This comment was copied over from the [ISM cypress.yml](https://github.com/opensearch-project/index-management-dashboards-plugin/blob/main/.github/workflows/cypress-workflow.yml#L25) that I used as a guide when creating the SAP yml, and it's present in the [alerting frontend](https://github.com/opensearch-project/alerting-dashboards-plugin/blob/main/.github/workflows/cypress-workflow.yml) as well, so it's an enhancement item for several frontend plugins at the moment. 

Created this issue to track this TODO https://github.com/opensearch-project/security-analytics/issues/170, and will revise the comment to include that link.